docs/core/deploying/trimming/trim-warnings/il2053.md

```diff
@@ -27,3 +27,7 @@ Value used in a `property` element in a custom attribute annotations file does n
   </assembly>
 </linker>
 ```
+
+## Obsolete
```

Comment:
I wonder if it would be better to add this in a Note block just before line 12.

> ![NOTE]
> This warning code is obsolete in .NET 7 and no longer produced by the tools.

Reply:
Good point - did that.